daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -80,6 +80,84 @@ The default value of `spec.metrics.http.increasedCardinality` is `true` in Dapr
 
 Setting `spec.metrics.http.increasedCardinality` to `false` is **recommended** to all Dapr users, to reduce resource consumption. The pre-1.13 behavior, which is used when the option is `true`, is considered legacy and is only maintained for users who have special requirements around backwards-compatibility.
 
+## HTTP Metrics Path Matching 
+
+Path matching allows you to manage and control the cardinality of HTTP metrics in Dapr. For details on how to set the cardinality in configuration see ({{< ref "configuration-overview.md#metrics" >}})  
```

Comment:
```suggestion
Path matching allows you to manage and control the cardinality of HTTP metrics in Dapr. This is an aggregation of metrics, so rather than having a metric for each event, you can reduce the number of metrics events and report an overall number.  For details on how to set the cardinality in configuration see ({{< ref "configuration-overview.md#metrics" >}})  
```